src/main/java/io/phasetwo/service/resource/MembersResource.java

```diff
@@ -112,4 +112,12 @@ private void canManage() {
               auth.getUser().getId(), organization.getName()));
     }
   }
+
+  private void canDelete(String userId) {
+    var seppuku = (userId == getUser().getId());
```

Comment:
Nits. I'd prefer:
1. we don't use java `var` even though we switched to java 11
2. we use a variable name that doesn't invoke suicide
I know we don't have a style guide, but thanks for changing these.

Reply:
1. Sounds good, not a Java dev, just winging this
2. I'm very sorry about that, didn't mean to commit it w/ that name. Was deep in hacking through Keycloak Themes / Java and dumped stress into variable names.

Reply:
No problem. Thanks for making the changes.

Reply:
I just dropped the var entirely and put the expression in the if block. LMK if you still want it as a semantically named boolean (like `isRemovingSelf`) or if you want the function named `canRemove` instead